File: program/check-consonant/CheckConsonant.js
function checkConsonants(str) {
    let consonant = '';
    if (str != 'a' && str != 'e' && str != 'i' && str != 'o' && str != 'u') {
        consonant = 'Consonant'
    }
    else {
        consonant = 'Is not a Consonant'
    }
    return consonant;
}

console.log("Given string is", checkConsonants("b"));

File: program/check-consonant/CheckConsonant.swift
import Swift
import Foundation

print("Please enter the letter:")
let given = String(readLine()!)
let consonants = "bcdfghijklmnpqrtvwxyz"

if consonants.contains(given) {
  print(given + " is consonant")
} else {
  print(given + " is not a consonant")
}
